src/main/java/org/spongepowered/common/mixin/core/tileentity/MixinTileEntityChest.java

```diff
@@ -98,7 +97,9 @@ public void onUpdate(CallbackInfo ci) {
         }
     }
 
-    @Inject(method = "openInventory", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/World;addBlockEvent(Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/block/Block;II)V"), cancellable = true)
```

Comment:
What is this actually changing? Nothing in this PR looks to really touch chests...